### ISSUE TITLE
Lab Report 4 Feedback

### ISSUE THREAD

**Issue by chanhenry54** [erwinmadjus/Week-2-Lab-Report#4]
Here is your feedback for Lab Report 4:

- [x] Provides link to their markdown-parse repository 
- [x] Provides link to the markdown-parse repository they reviewed

Snippet 1

- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Snippet 2

- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Snippet 3

- [x] Shows the test(s) / expected output in MarkdownParseTest.java
- [x] Shows the output of running test(s) for their implementation 
- [x] Shows the output of running for the implementation they reviewed
- [x] Answers the code change question about their program

Additional comments from your grader:
Heyo, I'm looking at your lab report 4 page, and it seems empty except for the links to the repositories. Were you able to complete it, or am I looking at the wrong page? This what I am looking at for reference: https://erwinmadjus.github.io/Week-2-Lab-Report/LabReport4-Week-8.html.

Feel free to comment below any questions about the rubric/feedback

--------

If you don't have all the checkmarks above, fix the issues described, and then submit a [regrade request on Gradescope](https://help.gradescope.com/article/8hchz9h8wh-student-regrade-request). Use this template for your regrade request:

- Link to this issue
- Link to your published lab report page

**Comment by erwinmadjus** [comment]
Hello, 

https://erwinmadjus.github.io/Week-2-Lab-Report/LabReport4-Week-8.html 

Sorry, my markdown-parse repository that I first had got corrupted / got deleted so I had to fork the professors markdown-parse file again and redo the entire thing.

I have completed the lab and here is the link to my website that contains the Week8 Lab 4 Lab Report. 


Would I still have to request for a regrade or will this be good? 


Thank you,

**Comment by chanhenry54** [comment]
Yup, please submit a regrade request whenever the option is available on Gradescope!

**Comment by erwinmadjus** [comment]
Alright will do, but can you please take a look to see if I have completed all the steps listed above?

Thanks!

**Comment by chanhenry54** [comment]
Yup, it looks good!

**Comment by erwinmadjus** [comment]
Thank you, I really appreciate it!
